catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -123,10 +126,12 @@ protected void onCreate(Bundle savedInstanceState) {
 		PreferenceManager.setDefaultValues(this, R.xml.preferences, true);
 		UtilUi.updateScreenWidthAndHeight(this);
 
+		Log.e(TAG, "Token : " + FirebaseInstanceId.getInstance().getToken());
+
```

Comment:
I think it definitely comes handy to log the token. Just as verbose, add `Firebase` to the text and move it to the CatroidApplication.create method.